cmd/kured/main.go

```diff
@@ -108,6 +110,18 @@ func sentinelExists() bool {
 		return false // unreachable; prevents compilation error
 	}
 }
+func forceRebootsentinelExists() bool {
```

Comment:
please run `go fmt` on the code

Reply:
thanx for the feedback, i have now run go fmt on the code, and fixed a bug where forcereboot was only performed if reboot was also requiered

Reply:
@awh What is your take on this , how does this fit the overall roadmap? do you eventually have other suggestions?